tests/FakeItEasy.Specs/InParametersSpecs.cs

```diff
@@ -79,9 +79,7 @@ public void SettingInParamDelegate(DelegateWithInParam fake, Exception exception
 
 #if LACKS_FAKEABLE_GENERIC_IN_PARAMETERS

```

Comment:
I thought about removing this section and just not testing, but I suppose there's a very small chance something will change. We might as well find out. But if you prefer, we could just to the `#else` part.